Add files via upload

Litviki opened the pull request:
Задание 3
Для каждой из 2 шард добавлено по 2 реплики 

shard1
shard1-mongodb2
shard1-mongodb3

shard2
shard2-mongodb2
shard2-mongodb3

Litviki committed:
Add files via upload

Задание 3
Для каждой из 2 шард добавлено по 2 реплики 

shard1
shard1-mongodb2
shard1-mongodb3

shard2
shard2-mongodb2
shard2-mongodb3

Litviki committed:
Add files via upload

Перенес настройки compose.yaml
запуск docker compose up -d проходит без ошибок
Последующие настройки в файле mongo_sharding_repl_cache_README.md

ivannizh commented:
Ошибки при запуске
![image](https://github.com/user-attachments/assets/b8981ab0-ce73-4856-994a-f7b63385cb46)
![image](https://github.com/user-attachments/assets/183d3a31-e027-44c6-b740-d73908c0b89c)
![image](https://github.com/user-attachments/assets/442229a1-7c16-4b1f-9d7a-5af6538eeaad)
![image](https://github.com/user-attachments/assets/7c918335-77cc-4f18-8b97-e79c0ec9dfda)

Litviki committed:
Add files via upload

Обновлен 
compose.yaml
mongo_sharding_repl_cache_README.md
redis.conf
1 redis
2 шарды
Для каждой шарды реплики

Litviki commented:
обновление
